utils/types.ts

```diff
@@ -1,6 +1,7 @@
 // Copyright 2023 the .NET Foundation
 
 import * as t from "io-ts";
+import * as tc from "io-ts-types";
```

Comment:
It looks like the changes in this file are not needed? Do we need to add `io-ts-types` (and/or monocle and newtype) as deps?